index.html

```diff
@@ -43,6 +43,7 @@
   <link rel="stylesheet" href="css/grid.css">
   <link rel="stylesheet" href="css/header.css">
   <link rel="stylesheet" href="css/section-1.css">
+  <link rel="stylesheet" href="css/section-2.css">
```

Comment:
Do not add another css file

Reply:
okay sir i will take care from next time.